### DISCUSSION TITLE
[HW3] Using `nn.MultiheadAttention` and `nn.LayerNorm`

### DISCUSSION THREAD

**Discussion by yuneg11** [seominjoon/kaist-ai605#140] [Q&A] [answered]
Could we use `nn.MultiheadAttention` and `nn.LayerNorm` already implemented in PyTorch?
Otherwise, should we implement it ourselves?

**Comment by seominjoon** [comment, marked as answer]
You can either use PyTorch's or copy from Annotated Transformer, whichever is easier for you.
